engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/BuiltinTypesTest.scala

```diff
@@ -344,10 +344,10 @@ class BuiltinTypesTest
     val requestId = UUID.randomUUID()
 
     val metadata = new Metadata
-    val idMain   = metadata.addItem(37, 19)
+    val idMain   = metadata.addItem(39, 19)
```

Comment:
I always wonder how people find these `addItem` parameters?

Reply:
Doesn't your IDE print line and character number ? :D